src/stories/TextInput.stories.js

```diff
@@ -57,6 +57,70 @@ Placeholder.args = {
     placeholder: 'My placeholder text',
 };
 
+const AutoGrow = Template.bind({});
+AutoGrow.storyName = 'Autogrow input';
+AutoGrow.args = {
+    label: 'Autogrow input',
+    name: 'AutoGrow',
+    placeholder: 'My placeholder',
+    autoGrow: true,
+    textInputContainerStyles: [{
+        minWidth: 150,
+    }],
+};
+
+const Prefixed = Template.bind({});
+Prefixed.storyName = 'Prefixed input';
+Prefixed.args = {
+    label: 'Prefixed input',
+    name: 'Prefixed',
+    placeholder: 'My placeholder text',
+    prefixCharacter: '@',
+};
+
+const WithFixedLabel = Template.bind({});
+WithFixedLabel.storyName = 'Always active label';
+WithFixedLabel.args = {
+    label: 'Active label input',
+    name: 'activelabel',
+    placeholder: 'My placeholder text',
+    forceActiveLabel: true,
+};
+
+const MaxLength = Template.bind({});
+MaxLength.storyName = 'Input with maxLength';
+MaxLength.args = {
+    label: 'Label',
+    name: 'inputmaxlength',
+    placeholder: 'My placeholder text',
+    maxLength: 50,
+};
+
+const HintErrorInput = (args) => {
+    const [error, setError] = useState('');
+    return (
+        <TextInput
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...args}
+            onChangeText={(value) => {
+                if (value && value.toLowerCase() === 'damn!') {
```

Comment:
Why `damn!`? 😄

Reply:
Just an example. :rofl:  It fits into `Damn! there is an error.` hint messages.

Reply:
😅 I think it's probably better to use something like `Oops!`